server/bin/pbench-server-activate-create-crontab

```diff
@@ -148,15 +148,6 @@ for role in $roles ;do
                 done
             done
             ;;
-        pbench-prep)
```

Comment:
Because we no longer have `pbench-server-prep-shim` at all, we remove the `pbench-prep` role entirely.